src/plot_api/plot_api.js

```diff
@@ -2765,10 +2799,13 @@ function makePlotFramework(gd) {
     // TODO: sort out all the ordering so we don't have to
     // explicitly delete anything
     fullLayout._glcontainer = fullLayout._paperdiv.selectAll('.gl-container')
-        .data([0]);
+        .data([{}]);
```

Comment:
what about this one here?

Reply:
that is the way `parcoords` is doing it's thing: https://github.com/plotly/plotly.js/pull/2159/files#diff-2fad23de82bf8c7625ea8e11436233b7R290
I was not really digging in why is it done this and not another way. Before it used own container for storing this data.

Reply:
Does it work ok with `data([0])`?

Reply:
@etpinard that is the point that parcoords puts `vm` property there via `extendFlat`, probably it is possible to find a better storage for it, I just did not figure out what.